### PR TITLE
Feature/to do column

### PR DESCRIPTION
**Pull request by rekloyd**
#27 - Task Project Link
Added the ToDoColumn component to show the column layout on page.

Also fixed App layout to be flex

### CODE REVIEW

**Review by JiajunQiuWu — APPROVED** [review]

**Review by JoseAlfredoMiguel — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+  const handleProjectSelect = (project: Project) => {
+    setSelectedProject(project);
+  };
+
   return (
     <>
-      <Sidebar/>
+    <div className="app_container">
+      <Sidebar onProjectSelect={handleProjectSelect} />
+      <ToDoColumn />
```
Comment (SaraVictorFernandez) [inline src/App.tsx:20]: Shouldn't ToDoColumn be inside ToDoWrapper?

File: src/components/ProjectComponent.tsx [hunk 2]
```diff
@@ -5,10 +5,10 @@ import React from 'react'
 import { Project } from '../types'
 
 
-export const ProjectComponent: React.FC<ProjectComponentProps> = ({project,deleteproject,editproject}: ProjectComponentProps) => {
+export const ProjectComponent: React.FC<ProjectComponentProps> = ({project,deleteproject,editproject, onClick}: ProjectComponentProps) => {
   return (
     <div className='Project'>
-      <p>{project.projectName}</p>
+      <p className='projectName' onClick={onClick}>{project.projectName}</p>
```
Comment (SaraVictorFernandez) [inline src/components/ProjectComponent.tsx:11]: projectName doesn't identify the project. It's better to send the id

File: src/components/ToDoWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Project } from "../types";
+
+export const ToDoWrapper: React.FC<ToDoWrapperProps> = ({ project }) => {
+  return (
+    <div>
+      <h1>Proyecto: </h1>
```
Comment (SaraVictorFernandez) [inline src/components/ToDoWrapper.tsx:6]: The column needs to be here

**Review by SaraVictorFernandez — CHANGES_REQUESTED** [review]

**Review by SaraVictorFernandez — APPROVED** [review]
Looks good to me

**Pull request merged**